frontend/packages/pipelines-plugin/src/components/pipelineruns/PipelineRunParametersForm.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Formik } from 'formik';
+import * as _ from 'lodash';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import PipelineRunParameters from './PipelineRunParameters';
+
+export interface PipelineRunFormProps {
+  obj: K8sResourceKind;
+}
+
+const PipelineRunForm: React.FC<PipelineRunFormProps> = ({ obj }) => {
```

Comment:
Update the function name to match with the filename

```suggestion
const PipelineRunParametersForm: React.FC<PipelineRunParametersForm> = ({ obj }) => {
```